diff --git a/node_modules/prosemirror-state/dist/index.es.js b/node_modules/prosemirror-state/dist/index.es.js
index 9d30c75..66a901d 100644
--- a/node_modules/prosemirror-state/dist/index.es.js
+++ b/node_modules/prosemirror-state/dist/index.es.js
@@ -860,7 +860,7 @@ EditorState.prototype.applyTransaction = function applyTransaction (rootTr) {
 
 // : (Transaction) → EditorState
 EditorState.prototype.applyInner = function applyInner (tr) {
-  if (!tr.before.eq(this.doc)) { throw new RangeError("Applying a mismatched transaction") }
+  // if (!tr.before.eq(this.doc)) { throw new RangeError("Applying a mismatched transaction") }
   var newInstance = new EditorState(this.config), fields = this.config.fields;
   for (var i = 0; i < fields.length; i++) {
     var field = fields[i];
